src/main/scala/org/scalastyle/Checker.scala

```diff
@@ -115,7 +115,8 @@ object Checker {
 class CheckerUtils(classLoader: Option[ClassLoader] = None) {
   private def comments(tokens: List[Token]): List[Comment] =
     tokens.flatMap { t =>
-      if (t.associatedWhitespaceAndComments == null) Nil else t.associatedWhitespaceAndComments.comments // scalastyle:ignore null
+      if (t.associatedWhitespaceAndComments == null) Nil
+      else t.associatedWhitespaceAndComments.comments // scalastyle:ignore null
```

Comment:
I believe the comment should be moved to the previous line?
Or refactor all that into:

```scala
Option(t.associatedWhitespaceAndComments).fold(ifEmpty = List.empty[Comment])(_.comments)
```

Reply:
> I believe the comment should be moved to the previous line?

good catch